README.md

```diff
@@ -428,6 +428,29 @@ for await (const movement of await account.movements.all()) {
 }
 ```
 
+### Webhook Signature Validation
+
+To ensure the authenticity of incoming webhooks from Fintoc, you should always validate the signature. The SDK provides a `WebhookSignature` class to verify the `Fintoc-Signature` header
+
+```javascript
+WebhookSignature.verifyHeader(
+    req.body,
+    req.headers['fintoc-signature'],
+    'your_webhook_secret'
+)
+```
+
+The `verifyHeader` method takes the following parameters:
+- `payload`: The raw request body as a string
+- `header`: The Fintoc-Signature header value
+- `secret`: Your webhook secret key (found in your Fintoc dashboard)
+- `toleranceSeconds`: Number of seconds to tolerate when checking timestamp (default: 300)
```

Comment:
a small nitpick

```suggestion
- `tolerance`: Number of seconds to tolerate when checking timestamp (default: 300)
```

in the Python SDK, this attribute is called `tolerance`. should we change it here? we could just ignore this, because in the python SDK these arguments are named arguments and here they are just positional